app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -177,8 +178,10 @@ public void deletePicture(final String filePath) {
     public void onDepictItemClicked(DepictedItem depictedItem) {
         if (depictedItem.isSelected()) {
             selectedDepictions.add(depictedItem);
+            DepictsFragment.selectedDepictedItem.add(depictedItem);
```

Comment:
We should ideally avoid static variables, can this not be done via callbacks?

Reply:
i thing no can you tell me other way

Reply:
It has to be done with "callbacks". You can basically search how to avoid static variables and how to use callbacks and you will find several online examples that shows you the required structure.